manim/mobject/geometry.py

```diff
@@ -395,6 +395,21 @@ def stop_angle(self):
 class ArcBetweenPoints(Arc):
     """
     Inherits from Arc and additionally takes 2 points between which the arc is spanned.
+
+    Example
+    --------------------
+    .. manim:: ArcBetweenPointsExample
+
+    class ArcBetweenPointsExample(Scene):
+        def construct(self):
+            circle = Circle(radius=2, stroke_color=GREY)
+            dot_1 = Dot(color=GREEN).move_to([2, 0, 0]).scale(0.5)
+            dot_1_text = Tex("(2,0)").scale(0.5).next_to(dot_1, RIGHT).set_color(BLUE)
+            dot_2 = Dot(color=GREEN).move_to([0, 2, 0]).scale(0.5)
+            dot_2_text = Tex("(0,2)").scale(0.5).next_to(dot_2, UP).set_color(BLUE)
+            arc1 = ArcBetweenPoints(start=2 * RIGHT, end=2 * UP, stroke_color=YELLOW)
+            self.add(circle, dot_1, dot_2, dot_1_text, dot_2_text)
+            self.play(Create(arc1))
```

Comment:
```suggestion
      class ArcBetweenPointsExample(Scene):
          def construct(self):
              circle = Circle(radius=2, stroke_color=GREY)
              dot_1 = Dot(color=GREEN).move_to([2, 0, 0]).scale(0.5)
              dot_1_text = Tex("(2,0)").scale(0.5).next_to(dot_1, RIGHT).set_color(BLUE)
              dot_2 = Dot(color=GREEN).move_to([0, 2, 0]).scale(0.5)
              dot_2_text = Tex("(0,2)").scale(0.5).next_to(dot_2, UP).set_color(BLUE)
              arc1 = ArcBetweenPoints(start=2 * RIGHT, end=2 * UP, stroke_color=YELLOW)
              self.add(circle, dot_1, dot_2, dot_1_text, dot_2_text)
              self.play(Create(arc1))
```

Reply:
I think one indent is missing here.
I hope that I made the right suggestion, maybe it needs one more